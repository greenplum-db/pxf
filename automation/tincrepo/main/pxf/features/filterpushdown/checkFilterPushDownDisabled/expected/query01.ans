-- start_ignore
-- end_ignore
-- @description query01 for PXF filter pushdown disabled case
SET gp_external_enable_filter_pushdown = off;
SET
SELECT * FROM test_filter WHERE  t0 = 'C' AND a1 = 2 ORDER BY t0, a1;
 t0 | a1 | b2 |  c3  | d4  | e5 | filtervalue
----+----+----+------+-----+----+-------------
 C  |  2 |    | 2.21 | CC  | CC | No filter
(1 row)

