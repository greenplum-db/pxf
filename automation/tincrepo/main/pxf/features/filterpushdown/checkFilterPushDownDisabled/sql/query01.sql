-- @description query01 for PXF filter pushdown disabled case

SET gp_external_enable_filter_pushdown = off;
SELECT * FROM test_filter WHERE  t0 = 'A' and a1 = 0 ORDER BY t0, a1;
