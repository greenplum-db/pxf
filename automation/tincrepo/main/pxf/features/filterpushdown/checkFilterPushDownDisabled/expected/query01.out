-- @description query01 for PXF Hive filter pushdown disabled case
SET gp_external_enable_filter_pushdown = off;
SELECT * FROM test_filter WHERE  t0 = 'A' and a1 = 0 ORDER BY t0, a1;
 t0 | a1 | b2 | filtervalue 
----+----+----+-------------
 A  |  0 | t  | No filter
(1 row)

